eva-server/src/main/java/uk/ac/ebi/eva/server/configuration/EvaPropertyConfiguration.java

```diff
@@ -0,0 +1,9 @@
+package uk.ac.ebi.eva.server.configuration;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ComponentScan(basePackages = {"uk.ac.ebi.eva.lib.config"})
```

Comment:
When I remove this class and add the component scan to the Application class the WSServer tests fail and return 401 errors. I created this class to do this component scan in the same manner as the [MongoEvaLibConfiguration](https://github.com/EBIvariation/eva-ws/blob/develop/eva-server/src/main/java/uk/ac/ebi/eva/server/configuration/MongoEvaLibConfiguration.java) and [JpaRepositoryConfiguration](https://github.com/EBIvariation/eva-ws/blob/develop/eva-server/src/main/java/uk/ac/ebi/eva/server/configuration/JpaRepositoryConfiguration.java) classes.